tools/ratbagctl.test.in

```diff
@@ -317,17 +317,17 @@ class TestRatbagCtlResolution(TestRatbagCtl):
         command = "resolution active set"
         r = self.launch_good_test("test_device resolution active get")
         self.assertEqual(int(r), 2)
-        r = self.launch_good_test("test_device " + command + " 0")
+        r = self.launch_good_test("test_device " + command + " 1")
         self.assertEqual(r, '')
         r = self.launch_good_test("test_device resolution active get")
-        self.assertEqual(int(r), 0)
+        self.assertEqual(int(r), 1)
         self.launch_fail_test(command)
         self.launch_fail_test("test_device " + command + " X")
         self.launch_fail_test("test_device " + command + " 1 X")
 
     def test_resolution_n_get(self):
         r = self.launch_good_test("test_device resolution 1 get")
-        self.assertEqual(r, "1: 1200x1300dpi (default)")
+        self.assertEqual(r, "1: 1200x1300dpi (active)")
```

Comment:
I fail to see how this is related.

Reply:
Me as well right now, but the output changes. Have to dig deeper here, but probably because Resolution 0 has zero dpi, which is btw a bad default value.

Reply:
Actually the first resolution wasn't default after all, default is the 0th resolution. Fixed the active part, I don't know what was wrong there but now it should be fine.